lib/chef/knife/ssh.rb

```diff
@@ -98,6 +98,10 @@ class Ssh < Knife
         :boolean => true,
         :default => true
 
+      option :sudo_password,
+        :long => "--sudo-password PASSWORD",
+        :description => "The password to use for 'sudo' commands. WARNING: using this option has limited security issues"
```

Comment:
The sudo password will show up in the ps listing.  The warning text could definitely be clearer.